review first: add wave to get realtime weather data (#9)
AdaGold/weather-report: README.md

Our goal is to create a fun, small weather app that focuses on displaying the weather.

Our weather app will set the weather using user interaction and get the weather from a 3rd party API, OpenWeather.

![Example weather app: The temperature reads 62, in yellow text. The selected dropdown for "Sky" is "Cloudy." There is a depiction of cloudy weather. The city name is "Hoboken." The header reads "Hoboken."](ada-project-docs/assets/cloudy-62.png)  

> _Fig. Example weather app: The temperature reads 62, in yellow text. The selected dropdown for "Sky" is "Cloudy." There is a depiction of cloudy weather. The city name is "Hoboken." The header reads "Hoboken."_

Reviewer: Some places in the curriculum/projects get a little out of hand with the captions, repeating the alt test as the caption. This is an antipattern that we should fix (screen readers are essentially presented redundant descriptions). This is low priority, but it would be great going forward if we could keep an eye out for situations like this and replace the caption with a shortened statement. Like here:

```suggestion
_Fig. Example weather app displaying the weather for Hoboken._
```

The detail is good for the alt text, but is very noisy for the caption.

You are required to use vanilla JavaScript for all parts of this project, including the optional enhancements.

> ## Axios

Reviewer 2: Will students have used Axios before this point? I'm wondering if it would be helpful to link to their docs in this section.

Reviewer 3: It's scheduled to be introduced the day after the project intro, but it would still be nice to link their docs.

> The only extra JavaScript libraries we should load is `Axios`.

Reviewer 2: ```suggestion
The only extra JavaScript library we should load is `Axios`.
```

To download the `axios` node module, run `npm install`

To include axios in your project, include the following script tag below the script tag linking `index.js`:

> - <script src="./node_modules/axios/dist/axios.min.js"></script>

Reviewer 2: We should escape the opening angle brackets or wrap the whole statement in a code block for it to display.
```suggestion
- `<script src="./node_modules/axios/dist/axios.min.js"></script>`
```

Wave 3:

1. An element that displays the city name

> 1. An element that contains a `<input type="text">` element, used to rename the city

Reviewer 3: ```suggestion
1. An element that contains an `<input type="text">` element, used to rename the city
```
